fmt/testdata/FunctionCallArgsStatement/original.sol

```diff
@@ -0,0 +1,37 @@
+interface ITarget {
+    function run() external payable;
+    function veryAndVeryLongNameOfSomeRunFunction() external payable;
+}
+
+contract FunctionCallArgsStatement {
+    ITarget public target;
+
+    function estimate() public returns (uint256 gas) {
+        gas = 1 gwei;
+    }
+
+    function veryAndVeryLongNameOfSomeGasEstimateFunction() public returns (uint256) {
+        return gasleft();
+    }
+
+    function value(uint256 val) public returns (uint256) {
+        return val;
+    }
+
+    function test() external {
+        target.run{ gas: gasleft(), value: 1 wei };
+
+        target.run{gas:1,value:0x00}();
+
+        target.run{ 
+                gas : 1000, 
+        value: 1 ether 
+        } ();
+
+        target.run{  gas: estimate(),
+    value: value(1) }(); 
+
+        target.run { value:
+        value(1 ether), gas: veryAndVeryLongNameOfSomeGasEstimateFunction() } ();
+    }
```

Comment:
after we fix determination of the need for the whitespace, i'll also add tests covering comments